src/cotletico_manager.py

```diff
@@ -74,16 +14,74 @@ def teams_from_json(filename: str):
         text = files.read()
         data = json.loads(text)
 
+    with open('../data/flags.json') as file:
+        flags = json.load(file)
+
     teams = list()
     for team_name in data.keys():
         players = list()
         for player_info in data[team_name]:
-            player = Player(**player_info)
-            players.append(player)
-        team = Team(team_name, players)
-        teams.append(team)
+            pl = player.Player(**player_info)
+            players.append(pl)
+            pl.statistics = dict({'injured': False, 'red': False})
+        tm = team.Team(team_name, players, flag=flags[team_name])
+        teams.append(tm)
     return teams
 
 
 teams = teams_from_json('../data/players.json')
 print(len(teams))
+
+
+def ideal_squad(team, formation='4-4-2'):
```

Comment:
Consider simplifying the method code.
Part of the logic can be moved to separate methods.